### ISSUE TITLE
Model Performance on Medical Video Datasets

### ISSUE THREAD

**Issue by Nano1337** [MCG-NKU/E2FGVI#28]
First off, your paper on an end-to-end framework for guided video completion is amazing! I used it on a medical task and it provided satisfactory results. However, it seems that the original iterative FGVC method performed better for my medical image-related task. Is there any way that I can use the pre-trained weights to fine-tune the model? If so, can you advise me as to which parts of the network I should freeze to preserve inpainting quality while allowing the network to familiarize itself with my domain-specific data? 

I also only have access to one V100 GPU through Google Colab, so I'm unable to use distributed training. Any help would be greatly appreciated!

**Comment by Paper99** [comment]
Only one V100 GPU also works well. You could set the batch size as 1. 

I don't have much experience with fine-tuning our model on other datasets.
Maybe it is possible to train all parts of the network with a small learning rate.
